### PR TITLE
Add more info to exceptions thrown from ComposedCursorLoader#loadInBackground()

### PR DESCRIPTION
**Pull request by plnice**

### CODE REVIEW

**Comment by sulewicz** [comment]
Super zmiana!

File: library/src/main/java/com/getbase/android/db/loaders/ComposedCursorLoader.java [hunk 1]
```diff
@@ -157,4 +161,22 @@ public void dump(String prefix, FileDescriptor fd, PrintWriter writer, String[]
     writer.print("mResult=");
     writer.println(mResult);
   }
+
+  private String getQueryData() {
+    final String delim = ", ";
```
Comment (chalup) [inline library/src/main/java/com/getbase/android/db/loaders/ComposedCursorLoader.java]: How about using `MoreObjects.toStringHelper(this)`?

File: library/src/main/java/com/getbase/android/db/loaders/ComposedCursorLoader.java [hunk 2]
```diff
@@ -157,4 +161,22 @@ public void dump(String prefix, FileDescriptor fd, PrintWriter writer, String[]
...
+        + Arrays.toString(mProjection)
+        + delim
+        + "mSelection="
+        + mSelection
+        + delim
+        + "mSelectionArgs="
+        + Arrays.toString(mSelectionArgs)
+        + delim
+        + "mSortOrder="
+        + mSortOrder;
```
Comment (chalup) [inline library/src/main/java/com/getbase/android/db/loaders/ComposedCursorLoader.java]: loader id might also be useful

File: library/src/main/java/com/getbase/android/db/loaders/ComposedCursorLoader.java [hunk 3]
```diff
@@ -157,4 +162,16 @@ public void dump(String prefix, FileDescriptor fd, PrintWriter writer, String[]
     writer.print("mResult=");
     writer.println(mResult);
   }
+
+  private String getStringRepresentation() {
```
Comment (chalup) [inline library/src/main/java/com/getbase/android/db/loaders/ComposedCursorLoader.java]: Why not `toString()`?

Comment (sulewicz) [inline library/src/main/java/com/getbase/android/db/loaders/ComposedCursorLoader.java]: Why not toStringSoGrumpyGeorgeWontComplain()?

**Pull request merged**